### DISCUSSION TITLE
Pytorch v2.x 整体进度（翻译+校对）

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/pytorch-doc-zh#583] [Announcements]
## 更新时间： 2023-11-17

> 翻译进度

- 翻译进度:  100%
- 校验进度: 5%
- 提交方式：看到哪里不对，可以提交PR就行优化，有奖励的！
- 提交PR：请见 [Github 入门指南](https://interview.apachecn.org/GitHub/)。
- 可以采用：Google翻译/ChatGPT + 个人校验 的模式
- 翻译的注意事项: <https://github.com/apachecn/pytorch-doc-zh/discussions/623>

> 翻译内容：

- ~~教程任务全集：<https://github.com/apachecn/pytorch-doc-zh/discussions/622>~~
- ~~文档任务全集：<https://github.com/apachecn/pytorch-doc-zh/discussions/625>~~

> 认领方式：

- 直接在`本页面`留言（其他的页面只是方便翻阅资料）

```
## 教程

- [别名](github地址) - QQ - 529815144  或  [别名](github地址) - WX - 529815144

>  Introduction to PyTorch

- [100%] Learn the Basics
- [30%] Quickstart
- [0%] Tensors
```
 
- 数字为 翻译进度 或 校验进度

> 奖励机制：

- 本次翻译奖励为：<https://github.com/apachecn/pytorch-doc-zh/discussions/583#discussioncomment-6272520>
- 组织奖励机制的模版：<https://github.com/orgs/apachecn/discussions/243>

**Comment by jiangzhonglian** [comment]
> 翻译事项：

翻译前，可以提前认领自己想翻译的页面，以免发生内容重复和冲突，😄

**翻译完后，一定要登记，后续会给大家发红包奖励的！**（自己手上全部章节翻译结束的，私聊片刻哈，领取红包）

> 翻译奖励：【更新时间：20230625】

文章按照：1元/章（可以采用Google翻译+自行校验的方式）+ 外加奖励（每10篇再奖励8元 / 6篇奖励4元，如果单篇内容超多，按照单篇 2/3倍计算）


> 翻译校验：【更新时间：20231117】

文章按照修正量来计算，正常可以给 0.5元/篇  + （每10篇再奖励4元 / 6篇奖励2元； 如果修改了较大，可以翻倍奖励）

如果大家对奖励有什么困惑，可以找我说，我随时调整

---

联系方式：微信 529815144（获取奖励私聊）

**Comment by lordqyxz** [comment]
本地测试页面：
```sh
pip install mkdocs mkdocs-material
mkdocs serve
```

**Comment by jiangzhonglian** [comment]
> 更新时间：2023-08-09

<https://pytorch.org/blog>  从1.2～2.0所有版本的新特性，我来更新

【完成，已经全部更新】 1.2 1.3 1.4 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12  1.13 2.0

**Comment by Daydaylight** [comment]
https://pytorch.org/tutorials/index.html
已完成：Introduction to PyTorch
进行中：Learning PyTorch

**Reply by Daydaylight** [reply]
6.26
https://pytorch.org/tutorials/beginner/basics/intro.html
https://pytorch.org/tutorials/beginner/basics/quickstart_tutorial.html
https://pytorch.org/tutorials/beginner/basics/tensorqs_tutorial.html

**Reply by Daydaylight** [reply]
6.29
[Datasets & DataLoaders](https://pytorch.org/tutorials/beginner/basics/data_tutorial.html#)
[Transforms](https://pytorch.org/tutorials/beginner/basics/transforms_tutorial.html)

**Reply by jiangzhonglian** [reply]
好兄弟，你的啥情况呢？

**Comment by sci-m-wang** [comment]
python API
torch.cuda https://pytorch.org/docs/stable/cuda.html     [done]
torch.distributed https://pytorch.org/docs/stable/distributed.html      [doing]

**Reply by jiangzhonglian** [reply]
好兄弟，你也注意一下，卡住了记得说一声：https://github.com/apachecn/pytorch-doc-zh/discussions/583#discussioncomment-7069414

**Comment by runzhi214** [comment]
- 贡献者信息
```
[@runzhi214](https://github.com/runzhi214): 270133721
```

6.29 - (已完成)
https://pytorch.org/tutorials/beginner/basics/buildmodel_tutorial.html (已完成)
https://pytorch.org/tutorials/beginner/basics/optimization_tutorial.html (已完成)
https://pytorch.org/tutorials/beginner/basics/autogradqs_tutorial.html (已完成)
https://pytorch.org/tutorials/beginner/basics/saveloadrun_tutorial.html (已完成)

6.30 -  (已完成)
Learning PyTorch章节:
- "LEARNING PYTORCH WITH EXAMPLES"  https://pytorch.org/tutorials/beginner/pytorch_with_examples.html  (已完成)
- "WHAT IS TORCH.NN REALLY?" https://pytorch.org/tutorials/beginner/nn_tutorial.html (已完成)

7.3 - 准备进行
Learning PyTorch章节:
- "DEEP LEARNING WITH PYTORCH: A 60 MINUTE BLITZ" https://pytorch.org/tutorials/beginner/deep_learning_60min_blitz.html

**Reply by jiangzhonglian** [reply]
好兄弟，你的啥情况呢？

**Reply by runzhi214** [reply]
这一篇没东西，就是个视频。在纠结是直接贴youtube的视频链接么？那样的倒是很容易。如果视频重发一份到内网的话我也不会弄～就搁置了

**Reply by jiangzhonglian** [reply]
直接贴 youtube 链接就行

**Comment by SwaggyZhang** [comment]
doc > torch

进行中

[https://pytorch.org/docs/stable/torch.html#](https://pytorch.org/docs/stable/torch.html#)

**Reply by jiangzhonglian** [reply]
好兄弟，我看到了，有一些内容太麻烦就不用翻译。

表格里面的太麻烦就别翻译，只翻译表格外面的说明就行
例如： 只翻译1的位置，2的位置不用翻译

<img width="810" alt="image" src="https://github.com/apachecn/pytorch-doc-zh/assets/9199175/307cd940-d998-4b8f-9404-8c2cf5a67b21">

**Comment by QX-N** [comment]
Model Optimization
进行中
https://pytorch.org/tutorials/beginner/profiler.html

**Comment by Fadegentle** [comment]
> ✅ ["Introduction to PyTorch on YouTube"](https://github.com/apachecn/pytorch-doc-zh/discussions/622) 

*   [Introduction to PyTorch - YouTube Series](https://pytorch.org/tutorials/beginner/introyt.html) ✅
*   [Introduction to PyTorch](https://pytorch.org/tutorials/beginner/introyt/introyt1_tutorial.html) ✅
*   [Introduction to PyTorch Tensors](https://pytorch.org/tutorials/beginner/introyt/tensors_deeper_tutorial.html) ✅
*   [The Fundamentals of Autograd](https://pytorch.org/tutorials/beginner/introyt/autogradyt_tutorial.html) ✅
*   [Building Models with PyTorch](https://pytorch.org/tutorials/beginner/introyt/modelsyt_tutorial.html) ✅
*   [PyTorch TensorBoard Support](https://pytorch.org/tutorials/beginner/introyt/tensorboardyt_tutorial.html) ✅
*   [Training with PyTorch](https://pytorch.org/tutorials/beginner/introyt/trainingyt.html) ✅
*   [Model Understanding with Captum](https://pytorch.org/tutorials/beginner/introyt/captumyt.html) ✅

**Reply by jiangzhonglian** [reply]
一共8篇：

```
1*1（短） + 5*2（中）  + 2 * 3（长） =  1 + 10 + 6 = 17  +  8 刺激奖励 = 25
```

**Comment by masteryi-0018** [comment]
## 教程

- [masteryi-0018](https://github.com/masteryi-0018) - QQ - 1536474741

> ✅ ["Deploying PyTorch Models in Production"](https://github.com/apachecn/pytorch-doc-zh/discussions/622)

- [100%] [Deploying PyTorch in Python via a REST API with Flask](https://pytorch.org/tutorials/intermediate/flask_rest_api_tutorial.html#)
- [100%] [Introduction to TorchScript](https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html)
- [100%] [Loading a TorchScript Model in C++](https://pytorch.org/tutorials/advanced/cpp_export.html)
- [100%] [(optional) Exporting a Model from PyTorch to ONNX and Running it using ONNX Runtime](https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html)
- [100%] [Real Time Inference on Raspberry Pi 4 (30 fps!)](https://pytorch.org/tutorials/intermediate/realtime_rpi.html)

**Reply by jiangzhonglian** [reply]
好兄弟，翻译完了，找我微信： 529815144

**Reply by jiangzhonglian** [reply]
` 0*1 + 3*2 + 2*2 = 10(+4 刺激奖励=14)`

**Comment by Noahs212** [comment]
进行中：[Reinforcement Learning (DQN) Tutorial](https://pytorch.org/tutorials/intermediate/reinforcement_q_learning.html)

**Reply by Noahs212** [reply]
- 已完成：[Reinforcement Learning (DQN) Tutorial](https://pytorch.org/tutorials/intermediate/reinforcement_q_learning.html)
- 进行中：[Reinforcement Learning (PPO) with TorchRL Tutorial](https://pytorch.org/tutorials/intermediate/reinforcement_ppo.html)
- 已完成：[TRAIN A MARIO-PLAYING RL AGENT](https://pytorch.org/tutorials/intermediate/mario_rl_tutorial.html)

**Reply by jiangzhonglian** [reply]
好兄弟，你的啥情况呢？

**Comment by SwaggyZhang** [comment]
我翻译了大部分，还有一些最近没时间就没搞。。



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年09月21日 19:26 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [apachecn/pytorch-doc-zh] Pytorch v2.0 整体进度（翻译+校对） (Discussion #583) |

好兄弟，你的啥情况呢？

**Comment by SwaggyZhang** [comment]


**Reply by jiangzhonglian** [reply]
没事，到时候给你多算奖励！

**Comment by SwaggyZhang** [comment]
当时想着力求还原英文文档，所以就全搞了



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年09月21日 19:41 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [apachecn/pytorch-doc-zh] Pytorch v2.0 整体进度（翻译+校对） (Discussion #583) |

没事，到时候给你多算奖励！

**Reply by jiangzhonglian** [reply]
抽空记得上传哈，😂到时候加我微信 529815144 领取奖励

**Comment by fangxiaoshen** [comment]
https://github.com/apachecn/pytorch-doc-zh/discussions/622

[Introduction to TorchRec](https://pytorch.org/tutorials/intermediate/torchrec_tutorial.html)
[Exploring TorchRec sharding](https://pytorch.org/tutorials/advanced/sharding.html)

**Reply by fangxiaoshen** [reply]
已完成

**Comment by fangxiaoshen** [comment]
正在翻译：["Multimodality"](https://github.com/apachecn/pytorch-doc-zh/discussions/622)

[TorchMultimodal Tutorial: Finetuning FLAVA](https://pytorch.org/tutorials/beginner/flava_finetuning_tutorial.html)

**Reply by fangxiaoshen** [reply]
已完成

**Comment by fangxiaoshen** [comment]
正在翻译：["Code Transforms with FX"](https://github.com/apachecn/pytorch-doc-zh/discussions/622)

[(beta) Building a Convolution/Batch Norm fuser in FX](https://pytorch.org/tutorials/intermediate/fx_conv_bn_fuser.html)
[(beta) Building a Simple CPU Performance Profiler with FX](https://pytorch.org/tutorials/intermediate/fx_profiling_tutorial.html)

**Reply by fangxiaoshen** [reply]
已完成

**Comment by fangxiaoshen** [comment]
领取任务：["Developer Notes"](https://github.com/apachecn/pytorch-doc-zh/discussions/625)

[CUDA Automatic Mixed Precision examples](https://pytorch.org/docs/stable/notes/amp_examples.html)
[Autograd mechanics](https://pytorch.org/docs/stable/notes/autograd.html)
[Broadcasting semantics](https://pytorch.org/docs/stable/notes/broadcasting.html)
[CPU threading and TorchScript inference](https://pytorch.org/docs/stable/notes/cpu_threading_torchscript_inference.html)
[CUDA semantics](https://pytorch.org/docs/stable/notes/cuda.html)
[Distributed Data Parallel](https://pytorch.org/docs/stable/notes/ddp.html)
[Extending PyTorch](https://pytorch.org/docs/stable/notes/extending.html)
[Extending torch.func with autograd.Function](https://pytorch.org/docs/stable/notes/extending.func.html)
[Frequently Asked Questions](https://pytorch.org/docs/stable/notes/faq.html)
[Gradcheck mechanics](https://pytorch.org/docs/stable/notes/gradcheck.html)
[HIP (ROCm) semantics](https://pytorch.org/docs/stable/notes/hip.html)
[Features for large-scale deployments](https://pytorch.org/docs/stable/notes/large_scale_deployments.html)
[Modules](https://pytorch.org/docs/stable/notes/modules.html)
[MPS backend](https://pytorch.org/docs/stable/notes/mps.html)
[Multiprocessing best practices](https://pytorch.org/docs/stable/notes/multiprocessing.html)
[Numerical accuracy](https://pytorch.org/docs/stable/notes/numerical_accuracy.html)
[Reproducibility](https://pytorch.org/docs/stable/notes/randomness.html)
[Serialization semantics](https://pytorch.org/docs/stable/notes/serialization.html)
[Windows FAQ](https://pytorch.org/docs/stable/notes/windows.html)

**Reply by jiangzhonglian** [reply]
卧槽！你这速度！ 666

**Reply by jiangzhonglian** [reply]
如果弄完了，可以找片刻要现金红包奖励

**Comment by SoTosorrow** [comment]
#670 更新了
[Deep Learning with PyTorch: A 60 Minute Blitz](https://pytorch.org/tutorials/beginner/deep_learning_60min_blitz.html) (末尾有四个跳转链接貌似中文版没有对应的文章所以就没翻译)
[TorchVision Object Detection Finetuning Tutorial](https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html)

**Comment by ryankang-proc** [comment]
amazing
